### PR TITLE
Dilpreet | Stage | Subscription Stage sanity Bugs

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by dvCodeWorld — DISMISSED** [review]

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html [hunk 1]
```diff
@@ -250,6 +252,50 @@
...
+                                                                        'feature-txt': !plan?.monthlyDiscountAmount > 0
+                                                                    }"
+                                                                >
+                                                                    <s class="strong">
+                                                                        <amount-field
+                                                                            [useGiddhCurrencyPipe]="false"
+                                                                            [spaceAfterCurrencySymbol]="false"
+                                                                            [amount]="plan?.monthlyAmount"
+                                                                            [currencySymbol]="plan?.currency?.symbol"
+                                                                            [currencyCode]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html]: why not sending currency code?

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html [hunk 2]
```diff
@@ -250,6 +252,50 @@
...
+                                                                        plan?.monthlyAmountAfterDiscount !== 0
+                                                                    "
+                                                                >
+                                                                    <span class="font-24 strong">
+                                                                        <amount-field
+                                                                            [useGiddhCurrencyPipe]="false"
+                                                                            [spaceAfterCurrencySymbol]="false"
+                                                                            [amount]="plan?.monthlyAmountAfterDiscount"
+                                                                            [currencySymbol]="plan?.currency?.symbol"
+                                                                            [currencyCode]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html]: why not sending currency code?

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html (hunk 1 above)
Comment (dilpreetsinghofficial) [reply apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html]: DONE

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html (hunk 2 above)
Comment (dilpreetsinghofficial) [reply apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html]: DONE

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**